contracts/periphery/SdexLpErc20.sol

```diff
@@ -33,7 +36,19 @@ contract SdexLpErc20 is ERC20Upgradeable, ISdexLpConduit {
         // token could be 0x0, which means the pair is against native ETH. quote
         // will never be 0x0 because native ETH will always be the base side of
         // the pair.
-        require(_quote != address(0) && _base != _quote && _quote > _base, "Invalid Token Pair");
+        require(
+            (_base == address(0) || _base.isContract()) 
+            && _base != address(this),
+            "Invalid Base Token");
+        require(
+            _quote.isContract() 
+            && _quote != address(this),
+            "Invalid Quote Token");
+        require(
+            _quote > _base, 
```

Comment:
The address of the quote token needs to be greater than the address of the base token?

Reply:
yes, quote token must always be greater than base token as per standard